pandas-stubs/core/frame.pyi

```diff
@@ -257,8 +275,8 @@ class DataFrame(NDFrame, OpsMixin):
     def to_records(
         self,
         index: _bool = ...,
-        columnDTypes: _str | dict | None = ...,
-        indexDTypes: _str | dict | None = ...,
+        column_dtypes: _str | dict[HashableT, _str | DtypeNp] | None = ...,
+        index_dtypes: _str | dict[HashableT, _str | DtypeNp] | None = ...,
```

Comment:
I think that `DTypeArg` is allowable for both of the above.  Docs say a "type" is OK.